src/wp-includes/script-modules.php

```diff
@@ -159,30 +159,11 @@ function wp_default_script_modules() {
 		/*
 		 * Build the WordPress Script Module ID from the file name.
 		 * Prepend `@wordpress/` and remove extensions and `/index` if present:
-		 *   - interactivity/index.min.js  => @wordpress/interactivity
-		 *   - interactivity/debug.min.js  => @wordpress/interactivity/debug
-		 *   - block-library/query/view.js => @wordpress/block-library/query/view
+		 *   - interactivity/index.min.js or interactivity/index.js => @wordpress/interactivity
+		 *   - block-library/query/view.min.js or block-library/query/view.js => @wordpress/block-library/query/view
 		 */
 		$script_module_id = '@wordpress/' . preg_replace( '~(?:/index)?(?:\.min)?\.js$~D', '', $file_name, 1 );
```

Comment:
https://github.com/WordPress/gutenberg/pull/72485 changed this line which I did _not_ copy, as far as I can tell the change is not significant: https://github.com/WordPress/gutenberg/pull/72485/files#r2560447901

Maybe it should be updated just for the sake of consistency:

```suggestion
		$script_module_id = '@wordpress/' . preg_replace( '~(?:/index)?\.(min\.)?js$~D', '', $file_name, 1 );
```